packages/ui/src/components/InputFields/SmartInputField/CountryCodeSelector/CountryCodeDropdown/index.tsx

```diff
@@ -0,0 +1,170 @@
+import type { Nullable } from '@silverhand/essentials';
+import { conditional } from '@silverhand/essentials';
+import { useCallback, useLayoutEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import ReactModal from 'react-modal';
+import { useDebounce } from 'usehooks-ts';
```

Comment:
https://github.com/juliencrn/usehooks-ts/blob/master/src/useDebounce/useDebounce.ts

shall we copy from this file with license? to reduce dependency

Reply:
Localized with license. Remove the dependency.